frontend/app/interfaces.ts

```diff
@@ -24,8 +24,9 @@ interface Tool {
   main_image_alt_text: string | null,
   privacy_agreement: string,
   canvas_placement_expanded: CanvasPlacement[],
-  support_resources: string
-  navigation_enabled: boolean
+  support_resources: string,
+  navigation_enabled: boolean,
+  sessionless_launch_url: string,
```

Comment:
I think that if the new workflow was adopted then we wouldn't need this here since it would get this url dynamically.